src/genai/extensions/localserver/schema.py

```diff
@@ -10,13 +12,18 @@ class TextGenerationCreateRequest(_TextGenerationCreateRequest):
     input: str
     model_id: str
     parameters: TextGenerationParameters = Field(default_factory=lambda: TextGenerationParameters())
+    data: Optional[PromptTemplateData] = None
+    input: Optional[str] = Field(None, examples=["How are you"], title="Input string")
+    moderations: Optional[ModerationParameters] = None
 
 
 class TextTokenizationCreateRequest(_TextTokenizationCreateRequest):
     """Following properties are required for local server"""
 
-    model_id: str
+    model_id: str = Field(...)
     parameters: TextTokenizationParameters = Field(default_factory=lambda: TextTokenizationParameters())
+    input: Optional[Union[str, list[str]]] = None
+    prompt_id: Optional[str] = None
 
 
 __all__ = ["TextGenerationCreateRequest", "TextTokenizationCreateRequest"]
```

Comment:
Even though inheritance is used, I had to import all the model's properties. When you do something like this.

```python
class TextTokenizationCreateRequest(_TextTokenizationCreateRequest):
    model_id: str = Field(...)
```

You won't be able to initiate it (Pydanti unknown error)—feel free to find out why.